prime-router/settings/prod/0163-sc-onboarding-to-up-receiver.yml

```diff
@@ -61,34 +61,45 @@
           truncateHl7Fields: []
           customLengthHl7Fields: {}
       jurisdictionalFilter:
-        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'SC') or (Bundle.entry.resource.ofType(Patient).address.state = 'SC')"
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state\
+      \ = 'SC') or (Bundle.entry.resource.ofType(Patient).address.state = 'SC')"
       qualityFilter:
+        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists()\
+      \ or Bundle.entry.resource.ofType(Specimen).collection.collected.exists()) or\
+      \ Bundle.entry.resource.ofType(serviceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists())"
+        - "(Bundle.entry.resource.ofType(Patient).address.line.exists() or Bundle.entry.resource.ofType(Patient).address.postalCode.exists()\
+      \ or Bundle.entry.resource.ofType(Patient).telecom.exists())"
         - "Bundle.entry.resource.ofType(MessageHeader).id.exists()"
         - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
         - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
         - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
         - "Bundle.entry.resource.ofType(Specimen).type.exists()"
-        - "(Bundle.entry.resource.ofType(Patient).address.line.exists() or Bundle.entry.resource.ofType(Patient).address.postalCode.exists() or Bundle.entry.resource.ofType(Patient).telecom.exists())"
-        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists()) or Bundle.entry.resource.ofType(serviceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists())"
       routingFilter: []
       processingModeFilter: []
       reverseTheQualityFilter: false
-      conditionFilter: []
+      conditionFilter:
+        - "%resource.where(code.coding.code in ('85478-6' | '85477-8' | '94500-6')).exists()\
```

Comment:
try using this as your conditionFilter.:

- "(%resource.interpretation.coding.code = 'A' and %resource.code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in ( '6142004' | '541131000124102' | '840539006')))"

This will catch any observations that are positive or detected and mapped to a condition of FLU, FLU A or COVID